src/otx/core/data/entity/base.py

```diff
@@ -57,13 +57,13 @@ class OTXDataEntity(Mapping):
     which can be go through the input preprocessing tranforms.
 
     :param task: OTX task definition
-    :param image: Image tensor which can have different type according to `image_type`
+    :param image: Image tensor or list of Image tensor which can have different type according to `image_type`
```

Comment:
Please add `image_type` for a list of Image tensors.

Reply:
Done by [3609c05](https://github.com/openvinotoolkit/training_extensions/pull/2737/commits/3609c05b5aa324e5678d0a549672ae990adae5f0)

Reply:
Please update docstring as well

Reply:
Done by [0a11d36](https://github.com/openvinotoolkit/training_extensions/pull/2737/commits/0a11d362ed1c384824cc8574f0f5c5021ededfb4)